geth_disabled.go

```diff
@@ -0,0 +1,20 @@
+// +build nimbus,!geth
+
+package main
+
+import (
+	"crypto/ecdsa"
+
+	status "github.com/status-im/status-protocol-go"
+	whispertypes "github.com/status-im/status-protocol-go/transport/whisper/types"
+)
+
+const noGethError = "executable needs to be built without -tags nimbus or with -tags geth"
```

Comment:
But you do `-tags "nimbus geth"` in `build-nimbus`.

Reply:
The only reason I do `-tags "nimbus geth"` is to maintain behavior equivalence with what exists in `develop` (i.e. you can choose to run Nimbus at runtime by passing `-nimbus` argument or default to geth). But this setup allows also building with `-tags nimbus` and that would still work fine (as long as you pass `-nimbus` to the console app).